node/tsconfig.json

```diff
@@ -5,10 +5,14 @@
         "module": "commonjs",
         "target": "es2015",
         "declaration": true,
-        "noImplicitAny": true,
+        //"noImplicitAny": true,
```

Comment:
Thanks for catching this - I didn't even notice this change :) I've re-added this rule, butinstead have fixed the violation instead